src/Datadog.AutoInstrumentation.NativeLoader/util.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+#include <filesystem>
+
+#include "pal.h"
+#include "string.h"
+
+using namespace datadog::shared::nativeloader;
+
+const std::string conf_filename = "loader.conf";
+const WSTRING cfg_filepath_env = WStr("DD_PROXY_CONFIGFILE");
+
+// Gets the profiler path
+static WSTRING GetProfilerPath()
+{
+    WSTRING profiler_path;
+
+#if BIT64
+    profiler_path = GetEnvironmentValue(WStr("CORECLR_PROFILER_PATH_64"));
+    if (profiler_path.length() == 0)
```

Comment:
Any particular reason why we try Core var first? Or just because we needed some order and this is as good as any? EIther way, perhaps this can be clarified in a comment, so that future generations know :)

Reply:
done in da1b96e